src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Printer, AreaCalculator  {
+    protected String color;

```

Comment:
it can be protected **but** this modifier also makes this field accessible inside the same package which is not consistent, so I would recommend making it private.
And in any way, it should have getters and setters